### PR TITLE
add overwrite-mode dropdowns (in place of replace checkboxes)

### PR DESCRIPTION
**Pull request by thedewi**
Input continues to use `Replace` field name and values in order to inherit previous version settings.

### CODE REVIEW

File: source/tasks/Metadata/task.json [hunk 1]
```diff
@@ -76,11 +76,16 @@
...
             "name": "Replace",
-            "type": "boolean",
-            "label": "Replace Existing",
-            "defaultValue": "False",
+            "type": "pickList",
+            "label": "Overwrite Mode",
+            "defaultValue": "false",
             "required": true,
-            "helpMarkDown": "If the package metadata already exists in the repository, the default behavior is to reject the new package metadata being pushed. Set this to 'True' to overwrite the existing package metadata."
+            "helpMarkDown": "If the package metadata already exists in the repository, the default behavior is to reject the new package metadata being pushed ('Fail if exists'). You can use the overwrite mode to 'Overwrite existing' or 'Ignore if exists'.",
```
Comment (hnrkndrssn) [inline source/tasks/Metadata/task.json]:
```suggestion
            "helpMarkDown": "Normally, if the same package metadata already exists on the server, the server will reject the push. This is a good practice as it ensures metadata isn't accidentally overwritten. Use this setting to override this behavior.
",
```

File: source/tasks/Push/PushV4/task.json [hunk 2]
```diff
@@ -54,11 +54,16 @@
...
             "name": "Replace",
-            "type": "boolean",
-            "label": "Replace Existing",
-            "defaultValue": "False",
+            "type": "pickList",
+            "label": "Overwrite Mode",
+            "defaultValue": "false",
             "required": true,
-            "helpMarkDown": "If the package already exists in the repository, the default behavior is to reject the new package being pushed. Set this flag to 'True' to overwrite the existing package."
+            "helpMarkDown": "If the package already exists in the repository, the default behavior is to reject the new package being pushed ('Fail if exists'). You can use the overwrite mode to 'Overwrite existing' or 'Ignore if exists'.",
```
Comment (hnrkndrssn) [inline source/tasks/Push/PushV4/task.json]:
```suggestion
            "helpMarkDown": "Normally, if a the same package already exists on the server, the server will reject the package push. This is a good practice as it ensures a package isn't accidentally overwritten or ignored. Use this setting to override this behavior.",
```

File: source/tasks/Metadata/task.json (hunk 1 above)
Comment (thedewi) [reply source/tasks/Metadata/task.json]: Okay, but I've added "or ignored", like Bamboo's message.

File: source/tasks/Push/PushV4/task.json (hunk 2 above)
Comment (thedewi) [reply source/tasks/Push/PushV4/task.json]: Okay, but I removed "a" typo.

**Pull request merged**